push-subscription-management/server.js

```diff
@@ -0,0 +1,61 @@
+var webPush = require('web-push');
+// collection of endpoints stored in memory (in production it should be
+// a database)
+var subscriptions = [];
```

Comment:
We could let the user enter a username as a "login"? But maybe that is a different recipe, where the push subscription is bound to the login status and has to be unsubscribed when the user logs out.